packages/app-mobile/utils/appDefaultState.ts

```diff
@@ -11,7 +11,6 @@ export const DEFAULT_ROUTE = {
 const appDefaultState: AppState = {
 	smartFilterId: undefined,
 	...defaultState,
-	sideMenuOpenPercent: 0,
```

Comment:
The `sideMenuOpenPercent` state was previously unused (only usage commented out).